compiler-plugin/src/main/kotlin/arrow/meta/quotes/ClassOrObject.kt

```diff
@@ -13,12 +13,112 @@ import org.jetbrains.kotlin.psi.psiUtil.getValueParameters
 import org.jetbrains.kotlin.psi.psiUtil.modalityModifierType
 import org.jetbrains.kotlin.psi.psiUtil.visibilityModifierType
 
+/**
+ * [classOrObject] is a function that intercepts all [KtClass] elements that [match]
```

Comment:
nit: Unintended newline?

Reply:
@JorgeCastilloPrz should I be letting it go as far as I want? I just press enter when I hit 80 characters out of habit. I don't normally work on docs so I'm not sure

Reply:
I'd suggest running the site locally then double check formatting.

Reply:
I will do so tonight after Kotlin User Group!

Reply:
Unable to run it locally - it looks like there are some compatibility issues in the gemfile for jekyll

```
Bundler could not find compatible versions for gem "rubygems":
  In Gemfile:
    rubygems (= 2.5.2.3)

    jekyll (~> 4.0.0) was resolved to 4.0.0, which depends on
      rubygems (>= 2.7.0)
```